region/crateria/central/Landing Site.json

```diff
@@ -551,6 +551,30 @@
         "A controlled release of the jump button works in place of the shortcharge, where the bottom of the thick dark line in the background at the top of the room works as a visual cue."
       ]
     },
+    {
+      "link": [3, 1],
+      "name": "Landing Site Big Jump Leave With Temporary Blue",
+      "notable": true,
+      "requires": [
+        "canCarefulJump",
+        "Morph",
+        "canSpeedball",
+        {"canShineCharge": {
+          "usedTiles": 30,
+          "openEnd": 2
+        }},
+        "canLongChainTemporaryBlue"
+      ],
+      "exitCondition": {
+        "leaveWithTemporaryBlue": {}
+      },
+      "note": [
+        "Use the runway near the Power Bomb room door to jump towards the Gauntlet Entrance room Bomb blocks and break them with a speedball;",
```

Comment:
Pretty sure you don't need a speedball and can bounce through, or unmorph at any time.
This is very similar to the "Landing Site Big Jump Blue Suit Bounce" notable.